### PR TITLE
Tdr 1719 judgment help guidance

### PR DESCRIPTION
**Pull request by MancunianSam**

### CODE REVIEW

File: app/views/judgment/judgmentHelp.scala.html [hunk 1]
```diff
@@ -0,0 +1,397 @@
...
+        </ul>
+
+        <h2 class="govuk-heading-m" id="before-you-begin">Before you begin</h2>
+
+        <h3 class="govuk-heading-s">Web browsers</h3>
+        <p class="govuk-body">
+          We do not support Internet Explorer (IE11). We advise you to use a different browser such as Edge, Chrome, Safari or Firefox to ensure you can use this service and all of its features.
+        </p>
+
+        <h3 class="govuk-heading-s">Secured access</h3>
```
Comment (TomJKing) [inline app/views/judgment/judgmentHelp.scala.html:56]: Should we add ids to the h3 headings as well, in case we want to link them?

**Review by TomJKing — COMMENTED** [review]
Looks good. Couple of minor comments. Would you be able to update the unit tests as well. See the FAQ PR for what I did for the FAQs unit tests.

File: app/views/judgment/judgmentHelp.scala.html (hunk 1 above)
Comment (MancunianSam) [reply app/views/judgment/judgmentHelp.scala.html:56]: :smile: Yeah. I've added an id to all header elements that didn't have one.

**Review by TomJKing — APPROVED** [review]
Looks good. Thanks

**Pull request merged**